SPECS/rt-tests/rt-tests.spec

```diff
@@ -102,8 +76,12 @@ make %{?_smp_mflags} DESTDIR=%{buildroot} prefix=%{_prefix} install
 /usr/share/man/man8/ssdd.8.gz
 /usr/share/man/man8/oslat.8.gz
 /usr/share/man/man8/get_cyclictest_snapshot.8.gz
+/usr/share/man/man8/determine_maximum_mpps.8.gz
 
 %changelog
+* Sat Jan 13 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 2.6-1
+- Auto-upgrade to 2.6 - none
```

Comment:
Any idea why "- none" is listed here and in the PR title? I think you can remove it from both places

Reply:
Maybe when the auto upgrade pipeline was run, there was no reason-for-update given.